services/core/jobs/delegates.js

```diff
@@ -26,7 +31,7 @@ module.exports = [
 		updateOnInit: true,
 		init: async () => {
 			logger.debug('Scheduling initial list update...');
-			await core.reloadDelegateCache();
+			await sleepAndExec(core.reloadDelegateCache);
```

Comment:
The delay is necessary only for the initialization scenario when the tables are getting created in the database. Since the block indexing also begins simultaneously, and the 'blocks' table doesn't exist yet, both this and the `init` method in blocks try to create the table, which results in an error and causes the indexing process to exit prematurely.